use_cases/eluc/data/constants.py

```diff
@@ -0,0 +1,85 @@
+from pathlib import Path
+
+import pandas as pd
+import regionmask
+
+# This should work if python path is set to ELUC
```

Comment:
do you mean to `use_cases/eluc` instead of `ELUC`?
```
This should work if PYTHONPATH is set to the root folder of this use case, "use_cases/eluc"
```